### ISSUE TITLE
Training Duration of ImageNet

### ISSUE THREAD

**Issue by wenhuiwei-ustc** [noahzn/Lite-Mono#64]
Hello, I would like to know how long your ImageNet training time is, and the GPU equipment used？

**Comment by noahzn** [comment]
Hi, the models were pre-trained on two A40 GPUs for 100 epochs, which took less than 3 days.

**Comment by noahzn** [comment]
@wenhuiwei-ustc  I'm closing this issue due to no response.

**Comment by wenhuiwei-ustc** [comment]
Thank you for your response!
Additionally, I would like to inquire about the approximate results of your ImageNet pre-training.

**Comment by noahzn** [comment]
Hi, I remember the accuracy being about 73% for Lite-Mono (3.1M), but I don't remember exactly.

**Comment by wenhuiwei-ustc** [comment]
Thank you for your response!
Furthermore, I would like to inquire about the learning rate and batch size of your ImageNet pre-training.

**Comment by noahzn** [comment]
The learning rate is scheduled in the script. I don't remember the exact batchsize that I used. It should be around 300.
